dockerfiles/init/modules/keycloak/files/che/email/text/footer.ftl

```diff
@@ -0,0 +1,9 @@
+--------------------------------------------------
```

Comment:
is it the official ascii logo of eclipse che ?

Reply:
That's just ascii logo of CHE

Reply:
yes that was more a question for @slemeur 

because we have tons of ascii rendering
http://patorjk.com/software/taag/#p=testall&t=Eclipse%20Che

Reply:
Yep - agree, I don't think the ascii logo is necessary in the plain text emails.

Reply:
@slemeur so, footer should be like this:
```
--------------------------------------------------
http://www.eclipse.org/che/
--------------------------------------------------
```
?

Reply:
+1